src/pages/Upload.js

```diff
@@ -596,7 +596,7 @@ const MetadataUploadPanel = _.flow(
       // Convert the table data structure back into a TSV, in case the user made changes
       const file = Utils.makeTSV([metadata.table.columns, ...metadata.table.rows])
       const workspace = Ajax().Workspaces.workspace(namespace, name)
-      await workspace.importFlexibleEntitiesFile(file)
+      await workspace.importFlexibleEntitiesFile(false, file)
```

Comment:
I preserved this as a non-async import, but am open to changing it to true

Reply:
where is this in the UI?

Reply:
I couldn't figure that out either! 
Isaac says:
> That's a whole separate upload pathway for covid stuff (I think?) that was built with external help
Brian R says:
> Correct. That page isn’t linked from anywhere. Presumably, those who have been invited to use it were given links.

So, I think we should keep it false for now to avoid messing up that page and then having to hotfix or rollback. After the fire is over, we could contact the external person or whoever owns that page and see if they want to convert the import there to async. How does that sound?

Reply:
I saw answers in Slack - agree to leave this exactly as-is for compatibility